content/influxdb/cloud/account-management/limits.md

```diff
@@ -0,0 +1,141 @@
+---
+title: InfluxDB Cloud limits
+description: >
+  InfluxDB Cloud has adjustable account limits and hard system limits.
+weight: 110
+menu:
+  influxdb_cloud:
+    parent: Account management
+    name: Limits
+products: [cloud]
+---
+
+InfluxDB Cloud has adjustable account limits per organization and hard global system limits. Review the following InfluxDB Cloud limits and errors to plan for your bandwidth needs:
+
+- [Account limits](#account-limits)
+- [Global limits](#global-limits)
+- [Limit errors](#limit-errors)
+
+<!--To estimate your projected usage costs, use the [InfluxDB Cloud pricing calculator](/influxdb/cloud/account-management/pricing-calculator/). -->
+
+## Account limits
+
+InfluxDB Cloud has adjustable limits applied per-account to reduce the chance of unexpected charges and protect the service for all users.
+
+_To request higher limits, reach out to [InfluxData Support](https://support.influxdata.com/)._
+
+### Rate limits
+
+{{% warn %}}
+Rate limits are accrued against a five-minute window.
```

Comment:
We should also mention that rate limits are applied to uncompressed payloads for writes.